### ISSUE TITLE
Update the 'X' in notQualify.html to redirect to the dashboard

### ISSUE THREAD

**Issue by TiCampbell** [Get-Your/GetFoco#57]
The current target is the webapp index page, which seems unintuitive. Since `notQualify.html` is only called from the dashboard ‘quick apply’ buttons, having the ‘X’ return to the dashboard seems more expected.

**Comment by penoptech** [comment]
@TiCampbell - Can we get rid of all of the logic related to the `notQualify` page? I ask because it looks like we don't actually display a quick apply button if the user's `AmiRange_max` is greater than program's `percentAmi`.

**Comment by TiCampbell** [comment]
Yes indeed! This is a relic from when we started the app with showing all programs for visibility, but now that we've gone with filtering by eligibility (via `AmiRange_max`), please remove anything that has to do with displaying to the user that they're not qualified for something.
